### PR TITLE
Feature/python3

### PR DESCRIPTION
**Pull request by Br3nda**
* Technical improvement.
 - upgrade to python 3.7

- - - -

These changes
- Change non-functional parts of this repository

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -72,14 +36,80 @@ jobs:
...
+          paths:
+            - /tmp/venv/aotearoa
+
+      - run:
+          name: Run tests
+          command: make test
+
+  build_python3:
+    docker:
+      - image: python:3.7
```
Comment (dlouise64) [inline .circleci/config.yml]: @Br3nda is this going to build python 3.6 or 3.7? Changelog says 3.6

Comment (Br3nda) [reply .circleci/config.yml]: good catch! i ended up copying 3.7 from Uruguay. I'll fix the changelog

**Comment by Br3nda** [comment]
it's unhappy on heroku.
https://openfisca-aotearoa-pr-98.herokuapp.com/spec

so i'm missing something

**Comment by Br3nda** [comment]
```
2018-11-11T20:36:02.054776+00:00 app[web.1]:     from openfisca_web_api.app import create_app

2018-11-11T20:36:02.054782+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/openfisca_web_api/app.py", line 19, in <module>

2018-11-11T20:36:02.054968+00:00 app[web.1]:     handle_import_error(error)

2018-11-11T20:36:02.054973+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/openfisca_web_api/errors.py", line 10, in handle_import_error

2018-11-11T20:36:02.055158+00:00 app[web.1]:     raise ImportError("OpenFisca is missing some dependencies to run the Web API: '{}'. To install them, run `pip install openfisca_core[web-api]`.".format(error))

2018-11-11T20:36:02.055169+00:00 app[web.1]: ImportError: OpenFisca is missing some dependencies to run the Web API: 'No module named 'flask''. To install them, run `pip install openfisca_core[web-api]`.```

**Comment by Br3nda** [comment]
!     Requested runtime (python-3.7.1) is not available for this stack (heroku-16).

I'll pick another

**Comment by Br3nda** [comment]
i've bumped us to heroku-18, and used python-3.7.0
i've bumped production to use heroku-18 on next deploy

**Comment by Br3nda** [comment]
It's all working on heroku now

**Pull request merged**